include/nuttx/lib/math.h

```diff
@@ -394,6 +394,14 @@ long double frexpl(long double x, int *exp);
 
 /* Trigonometric Functions **************************************************/
 
+void        sincosf(float, float *, float *);
```

Comment:
Please add `FAR` to added functions

Reply:
Is there any reason this one doens't have `FAR` added? `long double modfl (long double x, long double *iptr);`
Should I add FAR to all pointers in math.h?
e.g.
`long double frexpl(long double x, int *exp);`
`float       nanf(const char *tagp);`

Reply:
@pkarashchenko since the whole files doesn't add FAR, let's create a new patch add it in batch?